deploy.sh

```diff
@@ -5,7 +5,7 @@ stack_name="$1"
 
 if [ -z "$stack_name" ]
 then
-echo "😱 stack name expected as first argument, e.g. ./deploy di-ipv-cri-cic"
+echo "😱 stack name expected as first argument, e.g. ./deploy cic-cri-api-v1"
```

Comment:
could this be modified to `./deploy.sh cic-cri-api-v1` pls

Reply:
Sure..my understanding was this command is not being run as you can see its e.g.  Its for when you are deploying manually. Should i still change it?